### PR TITLE
fixes for lib64 libraries going in lib and deps updates for those packages

### PR DESCRIPTION
**Pull request by satmandu**
- Fixes for libvips, yasm, dropping libraries in `/usr/local/lib` on `x86_64`
- This led to rebuilding `libpng` (was linked to system libz on i686) & `libimagequant` (for i686), updating `qttools`, `poppler` (both deps for libvips)

Builds properly:
- [x] x86_64
- [x] i686
- [x] armv7l

### CODE REVIEW

File: packages/libvips.rb [hunk 1]
```diff
@@ -3,41 +3,46 @@
...
+    aarch64: '2d95674bfc951f232315ea0e93311d5982b9975c76f7f8d092f8d3fcfe6810b6',
+     armv7l: '2d95674bfc951f232315ea0e93311d5982b9975c76f7f8d092f8d3fcfe6810b6',
+       i686: '4bf07840cb30a398201160d0004530b909c48c691d21eb4bc35b2a4081c86f27',
+     x86_64: '3d2244043e9fd70c3a7429e98c825243cadac0e613a3c5443ee3a243721d8815'
   })
 
   depends_on 'cfitsio'
   depends_on 'fftw'
-  depends_on 'imagemagick'
+  depends_on 'imagemagick7'
```
Comment (uberhacker) [inline packages/libvips.rb:27]: Good call.  `imagemagick` is a fake package.

File: packages/poppler_data.rb [hunk 2]
```diff
@@ -3,26 +3,25 @@
...
-       i686: '45c2bfb58eae9b16e773a67d3c74a52f6d7096904d53495df6e710a1302bd41c',
-     x86_64: 'cb660cd7accbdb437feb6704133040c74e09f80065a3c2020959385e31f4eec4',
+  binary_sha256({
+    aarch64: '819a39d253b0e62d9e08f704e9d8d9bc551f48528393a82d988d87a64422ec63',
+     armv7l: '819a39d253b0e62d9e08f704e9d8d9bc551f48528393a82d988d87a64422ec63',
+       i686: '5982a173cc1a37ad97c0e5a79089e09ed9558acd11163d322b1b964dbf232d9b',
+     x86_64: '129a2d18480c9074f52a3bd2560f4fede057c065ec9823908c12f62b7c42bf20'
   })
 
-  depends_on 'poppler'
```
Comment (uberhacker) [inline packages/poppler_data.rb:24]: Good call.  Gets rid of dependency cycle.

File: packages/qttools.rb [hunk 3]
```diff
@@ -3,34 +3,35 @@
...
   def self.build
-    system 'qmake && make'
+    # Note: This MAY NOT COMPILE with llvm installed on x86. Install dependencies,
+    # disable depends, and uninstall llvm before attempting compile.
+    system 'qmake -Wall -early QMAKE_CFLAGS=-flto -early QMAKE_CXXFLAGS=-flto && make'
   end
 
   def self.install
-    FileUtils.mkdir_p "#{CREW_DEST_LIB_PREFIX}"
+    FileUtils.mkdir_p CREW_DEST_LIB_PREFIX.to_s
```
Comment (uberhacker) [inline packages/qttools.rb:34]: No need to convert to string.

Comment (satmandu) [reply packages/qttools.rb:34]: I need to find the rule that rubocop uses to lint these files and do this and disable that rule.

**Pull request merged**